aptos-move/framework/aptos-framework/sources/transaction_context.move

```diff
@@ -1,4 +1,58 @@
 module aptos_framework::transaction_context {
+
+    use std::features;
+
+    /// create_unique_address uses this for domain separation within its native implementation
+    #[test_only]
+    const DERIVE_UUID_ADDRESS_SCHEME: u8 = 0xFB;
```

Comment:
the point of having this and the test in object.move is to make sure that it is domain separated from other object methods correctly - and to make sure nobody adds a new domain separation that reuses 0xFB.

i.e. this constant needs to be next to OBJECT_DERIVED_SCHEME, OBJECT_FROM_GUID_ADDRESS_SCHEME etc.

Reply:
Done.